notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1593_force_actuator_error_elevation.ipynb

```diff
@@ -0,0 +1,514 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    df_primary_FA_error = await client.select_time_series(
</pre><p>Instead of using <code>await client.select_time_series</code> consider using <code>getEfdData</code> which you have imported. The function does basically the same without needing the <code>await</code>.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/148/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Indeed, I think I copy-pasted something very old because many of the options here I have been using in my more recent notebooks.

Reply:
<p>Updated</p>